worlds/tits_the_3rd/client/client.py

```diff
@@ -253,6 +269,21 @@ async def tits_the_3rd_watcher(ctx: TitsThe3rdContext):
             logger.info(f"Game Start Dectected. Setting up AP Verification Hook")
             ctx.game_interface.write_world_player_identifier(ctx.world_player_identifier)
             logger.info(f"AP Verification Hook Set. You can now save and resume the game for this AP Seed")
+            for character_item in ctx.items_received[:2]:  # The first 2 items are guarantee to be characters
+                character_item_id = character_item.item
+                logger.info(character_item_id)
```

Comment:
Is this a leftover log statement? Thinking we don't need to bloat the logs with this.

Reply:
yeah it is a leftover from testing